apis/src/main/kotlin/org/yapp/apis/auth/service/AuthServiceImpl.kt

```
@@ -0,0 +1,113 @@
+            provider = user.providerType.name
+        )
+    }
+
+    override fun getUserIdFromAccessToken(accessToken: String): Long {
+        if (!jwtTokenService.validateToken(accessToken)) {
+            throw AuthException(AuthErrorCode.INVALID_ACCESS_TOKEN)
+        }
+
+        try {
+            return jwtTokenService.getUserIdFromToken(accessToken)
+        } catch (e: JwtException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)